ci/test-clang-docker.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ ! -d /repo ]]; then
+    echo "[-] This script needs to be run within Docker! Use test-clang.sh instead."
+    exit 1
+fi
+
+clang=/repo/toolchain/bin/clang
+hello_c=/repo/llvm-project/hello.c
+hello_cpp=/repo/llvm-project/hello.cpp
+hello_exe=/tmp/hello
+host_arch=$(uname -m)
+
+# Print clang information, which makes sure it can run
+echo "[+] Testing 'clang --version'"
+"$clang" --version
+
+case "$(source /usr/lib/os-release; echo "$ID")" in
```

Comment:
do other distros have `/usr/lib/os-release` or just fedora?

Reply:
I think I answered this above, let me know if you have follow up questions!